### PR TITLE
DOC-1973: TinyMCE 6.6 Enterprise Documentation release

### PR DESCRIPTION
**Pull request by brian-forte**
DOC-1973: TinyMCE 6.6 Enterprise Documentation release

Changes:
* TinyMCE 6.6 Enterprise Documentation release.

Pre-checks:
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by ShiridiGandham — APPROVED** [review]
LGTM

File: changelog.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ Changes to the TinyMCE documentation are documented in this file.
 
 The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+### 2023-07-21
+
+- DOC-2093: the TinyMCE 6.6 Release notes.
```
Comment (kemister85) [inline changelog.md:9]:
```suggestion
- DOC-2093: The TinyMCE 6.6 Release notes.
```

File: modules/ROOT/examples/live-demos/ai-demo/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+tinymce.init({
+  selector: 'textarea#ai',
+  plugins: [
+    "advlist", "anchor", "autolink", "charmap", "code", "fullscreen", 
+    "help", "image", "insertdatetime", "link", "lists", "media", 
+    "preview", "searchreplace", "table", "visualblocks", "ai",
+    ],
+  height: 1200,
+  toolbar: "aidialog | undo redo |link image accordion | styles | bold italic underline strikethrough | align | bullist numlist"
```
Comment (kemister85) [inline modules/ROOT/examples/live-demos/ai-demo/index.js:11]:
```suggestion
  toolbar: "aidialog | undo redo |link image accordion | styles | bold italic underline strikethrough | align | bullist numlist",
```

File: modules/ROOT/pages/6.6-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+=== AI Assistant plugin toolbar items added to the default toolbar and AI Assistant plugin menu items added to the default menu bar
+//#TINY-9939
+
+With the release of the new xref:new-premium-plugin[AI Assistant Premium plugin], {productname}’s default menus and toolbars have been updated.
+
+If the AI Assistant plugin is added to a {productname} configuration, by default:
+
+. the _AI dialog_ and _AI shortcuts_ menu items will appear in the {productname} *Tools* menu; and
+
```
Comment (kemister85) [inline modules/ROOT/pages/6.6-release-notes.adoc:147]:
```suggestion
```

File: modules/ROOT/pages/6.6-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+As of the {productname} 6.6 update, the component has a minimum height of 200px, improving its out-of-the-box utility as a display component, particularly in medium dialogs.
+
+For more information about this component, see the xref:dialog-components.adoc#iframe[`iframe` section] of the Dialog components documentation.
+
+For more information about dialogs, see the xref:dialog-configuration.adoc[Dialog configuration] documentation.
+
+
+[[change]]
+== Change
```
Comment (kemister85) [inline modules/ROOT/pages/6.6-release-notes.adoc:198]: @brian-forte This is broken.
```suggestion
[[changes]]
== Changes
```

**Review by kemister85 — APPROVED** [review]

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 5]
```diff
@@ -68,6 +68,19 @@ Each of the event callback functions - `+onAction+`, `+onSubmit+`, `+onCancel+`,
...
+
+include::partial$configuraton/dialog_align.adoc[]
+
+include::partial$configuraton/dialog_border.adoc[]
+
+include::partial$configuration/dialog_buttons.adoc[]
+
+include::partial$configuraton/dialog_persistent.adoc[]
+
+include::partial$configuraton/dialog_streamContent.adoc[]
```
Comment (brian-forte) [inline modules/ROOT/pages/dialog-configuration.adoc:82]:
```suggestion
include::partial$configuration/dialog_persistent.adoc[]

include::partial$configuration/dialog_streamContent.adoc[]
```
Typo correction in include statements.

File: modules/ROOT/pages/dialog-configuration.adoc [hunk 6]
```diff
@@ -68,6 +68,19 @@ Each of the event callback functions - `+onAction+`, `+onSubmit+`, `+onCancel+`,
 
 Any callback function that is not passed a `+details+` object assumes that the dialog will only contain one component which can trigger it or that it does not matter if the function is triggered by multiple components. For example, `+onSubmit+` is only triggered when a user clicks on a *Submit* type footer button, and {productname} assumes that a dialog will only have one *Submit* type button. In comparison, `+onCancel+` and `+onClose+` are both triggered by clicking the `+X+` button in the top right of a dialog or by clicking a *Cancel* type footer button. These two buttons have the same functionality, and therefore {productname} does not differentiate between them.
 
+[[configuration-parameters]]
+== Configuration parameters
+
+include::partial$configuraton/dialog_align.adoc[]
+
+include::partial$configuraton/dialog_border.adoc[]
```
Comment (brian-forte) [inline modules/ROOT/pages/dialog-configuration.adoc:76]:
```suggestion
include::partial$configuration/dialog_align.adoc[]

include::partial$configuration/dialog_border.adoc[]
```

File: modules/ROOT/pages/ai.adoc [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+For example, interfacing with the OpenAI Completions API:
+
+[source,js]
+----
+// This example stores the API key in the client side integration. This is not recommended for any purpose.
+// Instead, an alternate method for retrieving the API key should be used.
+const api_key = '<INSERT_API_KEY_HERE>';
+
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
```
Comment (kemister85) [inline modules/ROOT/pages/ai.adoc:43]:
```suggestion
  selector: 'textarea',  // Change this value according to your HTML
```

**Review by kemister85 — APPROVED** [review]

**Pull request merged**